change/@fluentui-react-charting-cfa801e0-6573-413b-bb65-4c94f5e8ddbe.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Sankey chart changes for the new design",
+  "packageName": "@fluentui/react-charting",
+  "email": "ankityadav@microsoft.com",
```

Comment:
The behavior in dark mode seems not correct. The borders are still in dark making them invisible.

Reply:
As discussed, I will get some information from design team regarding this.

Reply:
The links are still showing light background in dark mode.

Reply:
This was working properly under selected dark theme from system, but when we select the dark theme on demo site there it was still showing links in light background. This is fixed now.